backend/tests/test_event_creation.py

```diff
@@ -0,0 +1,372 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from sqlalchemy.orm import Session
+
+from capellacollab.projects.users.crud import add_user_to_project
+from capellacollab.projects.users.models import (
+    ProjectUserPermission,
+    ProjectUserRole,
+)
+from capellacollab.users.crud import (
+    create_user,
+    get_user_by_id,
+    get_user_by_name,
+)
+from capellacollab.users.events.models import (
+    DatabaseUserHistoryEvent,
+    EventType,
+)
+from capellacollab.users.models import DatabaseUser, Role
+
+reason: str = "TestReason"
+
+
+def test_create_admin_user_by_system(db):
+    user: DatabaseUser = get_user_by_name(db, "your-username")
```

Comment:
Yes, I normally use uuids for names, but this is the name of the default admin that is created when the backend starts. (i.e., I don't create it here, I just get the default admin). But I agree with you that using static names like this is not so good, and this test could be replaced with one where we mirror the user creation via login (since this is much more work, I will keep the current test for now).